signing_service/signing_service/utils.py

```diff
@@ -20,7 +20,7 @@ def is_valid_public_key(key: bytes) -> bool:
         return False
 
 
-def is_valid_private_key(key: str) -> bool:
+def is_private_key_valid(key: str) -> bool:
```

Comment:
I cannot find unit tests for this helper function... Are you sure they are there? ;)

Reply:
@dybi Sorry I was wrong, there no unit test for this function but for only function that uses it, which is `validate_arguments`, please tell me if these are enough (I think adding unit will be redundant)

```python
def test_that_signing_service__validate_arguments_should_raise_exception_on_wrong_length_of_ethereum_private_key(self):
    self.signing_service.ethereum_private_key = self.ethereum_private_key[:-1]

    with self.assertRaises(SigningServiceValidationError):
        self.signing_service._validate_arguments()

def test_that_signing_service__validate_arguments_should_raise_exception_on_wrong_characters_in_ethereum_private_key(self):
    self.signing_service.ethereum_private_key = self.signing_service.ethereum_private_key[:-1] + 'g'

    with self.assertRaises(SigningServiceValidationError):
        self.signing_service._validate_arguments()
```

Reply:
Added as discussed, done :+1: